.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,24 @@
+name: Node CI
```

Comment:
This file is named `nodejs.yml`, and there are a few other mentions of Node throughout this file. Is that just leftover from copy/paste? If so, can you remove them? I'm really hoping that we can end up landing this in Node, but it should also be able to run anywhere else that libuv can.

Reply:
It's copied from one of my Node.js project, I will try to change it :)